libnetwork/drivers/windows/windows.go

```diff
@@ -285,6 +286,12 @@ func (d *driver) CreateNetwork(id string, option map[string]interface{}, nInfo d
 		return fmt.Errorf("Unknown generic data option")
 	}
 
+	if v, ok := option[netlabel.EnableIPv4]; ok {
+		if enable_IPv4, ok := v.(bool); ok && !enable_IPv4 {
+			return errors.New("IPv4 cannot be disabled on Windows")
```

Comment:
Should this one also return a typed error (does this error make its way back to the API? Without an errdefs for it, it will return a `500` status.

Reply:
Yes, I think so ... like the error immediately above, the two immediately below, and probably some more.

Reply:
Oh, the ones below are typed - it's just the one above.

Reply:
I've changed the new one into an `errdefs.ErrInvalidParameter`.